src/haz3lweb/app/editors/stepper/StepperBase.re

```diff
@@ -426,22 +431,23 @@ module Update = {
       ) => {
     switch (step_kind) {
     | SingleStep(m) =>
-      calculate_single_step(~settings, expr, state, m, hidden, editor)
+      calculate_single_step(~settings, expr, ctx, state, m, hidden, editor)
```

Comment:
Why do we need to send in `ctx` here and elsewhere? Unless I'm missing it, it doesn't seem to get used.